internal/users/locking/locking.go

```diff
@@ -3,6 +3,7 @@ package userslocking
 import (
```

Comment:
```diff
-users/locking: Fail if users lock is not acquired in enough time
+users/locking: Fail if acquiring lock takes too long

libc by default errors out if the lock is not acquired in 15 seconds, we
-replicate the same to ensure that authd won't never hang indefinitely if
+replicate the same to ensure that authd will never hang indefinitely if
-the libc alarm mechanism doesn't work properly (as it may not be handled
+the libc alarm mechanism doesn't work properly (we've seen it not work
as expected when go runtime is involved).